app/controllers/user/crates.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+import PaginationMixin from '../../mixins/pagination';
+
+const { computed } = Ember;
+// TODO: reduce duplicatoin with controllers/crates
```

Comment:
small note here; eventually, controllers are going away in Ember, so this might be extra-fine for now :)